src/cephadm/cephadm.py

```diff
@@ -5422,12 +5422,20 @@ def prepare_ssh(
     if not ctx.skip_monitoring_stack:
         for t in ['ceph-exporter', 'prometheus', 'grafana', 'node-exporter', 'alertmanager']:
             logger.info('Deploying %s service with default placement...' % t)
-            cli(['orch', 'apply', t])
+            try:
+                cli(['orch', 'apply', t])
+            except Exception:
```

Comment:
Can we be more specific with the exception being caught? Catching all `Exception`s is a wide net and could mask other issues.

Reply:
I just checked the sources. call_throws will raise RuntimeError if call returns a non-success return code, so catching RuntimeError alone should be sufficient.

Reply:
swapped to only catching `RuntimeError`